Bottom: 8ed09141b3428276f37b157b83cc615a7e48d5d5
Top:    050591f89ab1109cc2ee5771fa46822a788676ba
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:56:05 +0000

Refresh of mark-segments-as-not-round

---

diff --git a/src/polygon1.c b/src/polygon1.c
index cc4419b..32d0ef7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,7 +39,6 @@
       are marked
 */
 
-#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -939,7 +938,6 @@ insert_vertex_in_seg (struct info *i, struct seg *s, Vector v, double param)
   DEBUGP ("new intersection on segment \"i\" at %#mD\n", v[0], v[1]);
 #endif
   i->node_insert_list = prepend_insert_node_task (i->node_insert_list, s, new_node, param);
-  assert (!s->intersected);
   s->intersected = 1;
   return true;
 }
